diff --git a/cfg/sky130.yaml b/cfg/sky130.yaml
index 595cce4..1dcd24a 100644
--- a/cfg/sky130.yaml
+++ b/cfg/sky130.yaml
@@ -106,5 +106,7 @@ tt:
         9:  1192120
         10:  976120
         11:  760120
+  huge_tiles:
+    mux_id: [ 19 ]
   logo:
     size: 100000
diff --git a/py/tt/layout.py b/py/tt/layout.py
index f01cb0b..c5db804 100644
--- a/py/tt/layout.py
+++ b/py/tt/layout.py
@@ -30,7 +30,7 @@ class Layout:
 		self.hspine_layout()
 		self.vspine_layout()
 		self.ctrl_layout()
-		self.analog_layout()
+		self.mux_layout()
 
 	def _align(self, v, layer, dir_, ceil=False):
 		# Grab config data for tracks / sites
@@ -514,7 +514,7 @@ class Layout:
 		self.ply_ctrl_io_top.update( spread(tr_pads, limit_right, self.glb.ctrl.width) )
 		self.ply_ctrl_io_bot.update( spread(br_pads, limit_right, self.glb.ctrl.width) )
 
-	def analog_layout(self):
+	def mux_layout(self):
 		# Generate list of masked muxes
 		self.mux_mask = 0
 
@@ -523,6 +523,10 @@ class Layout:
 				for mux_id in grp['mux_id']:
 					self.mux_mask |= 1 << mux_id
 
+		if hasattr(self.cfg.tt, 'huge_tiles'):
+			for mux_id in self.cfg.tt.huge_tiles.mux_id:
+				self.mux_mask |= 1 << mux_id
+
 	def mux_exists(self, mux_id):
 		# Check if a given mux exists or is masked for some reason
 		return not ((self.mux_mask >> mux_id) & 1)
diff --git a/rtl/tt_defs.vh.mak b/rtl/tt_defs.vh.mak
index 6effdc6..f276af1 100644
--- a/rtl/tt_defs.vh.mak
+++ b/rtl/tt_defs.vh.mak
@@ -22,5 +22,8 @@ if hasattr(cfg.tt, 'analog'):
 	for grp in cfg.tt.analog:
 		for mux_id in grp['mux_id']:
 			mux_mask |= 1 << mux_id
+if hasattr(cfg.tt, 'huge_tiles'):
+	for mux_id in cfg.tt.huge_tiles.mux_id:
+		mux_mask |= 1 << mux_id
 %>
 `define TT_MUX_MASK ${cfg.tt.grid.y}'h${'{:X}'.format(mux_mask)}
